ebmdatalab/generate-ebmdatalab-stats.py

```diff
@@ -97,35 +186,104 @@ def main(args):
+    single_countries_codes = pd.DataFrame(single_countries,
+                                          index=single_countries).join(country_codes)["iso_a3"]
+    single_countries_codes = single_countries_codes.dropna()
+    single_countries_counts = single_countries_codes.value_counts()
+    multi_countries_codes = \
+        pd.DataFrame(multi_countries.str.strip(),
+                     index=multi_countries.str.strip()).join(country_codes)["iso_a3"]
+    multi_countries_codes = multi_countries_codes.dropna()
+    multi_countries_counts = multi_countries_codes.value_counts()
+    all_counts = single_countries_counts.to_frame(name='single_countries_counts').\
+        merge(multi_countries_counts.to_frame(name='multi_countries_counts'),
+              how="outer",
+              left_index=True,
+              right_index=True)
+
+    # Map frequency data onto the geopandas geographical data for units with ISO code
+    # geopandas uses -99 as N/A for this field
+    # We don't need to evaluate Antarctica
+    countries_mapping = geopandas.read_file(geopandas.datasets.get_path('naturalearth_lowres'))
+    countries_mapping = lowres_fix(countries_mapping)
+    countries_mapping = countries_mapping[(countries_mapping.name != "Antarctica") &
+                                          (countries_mapping.iso_a3 != "-99")]
+    countries_mapping = countries_mapping.merge(all_counts,
+                                                how="left",
+                                                right_index=True,
+                                                left_on="iso_a3")
+
+    # Generate two-part choropleth of world map with # of clinical trials counted
+    fig, (ax1, ax2) = plt.subplots(nrows=2, ncols=1, figsize=(20, 16))
+    fig.patch.set_visible(False)
+    ax1.axis('off')
+    ax2.axis('off')
+    countries_mapping.boundary.plot(ax=ax1, edgecolor="black")
+    countries_mapping.plot(column='single_countries_counts', ax=ax1, legend=True)
+    ax1.set_title("Number of Single-Country Clinical Trials Recruiting by Country")
+    countries_mapping.boundary.plot(ax=ax2, edgecolor="black")
+    countries_mapping.plot(column='multi_countries_counts', ax=ax2, legend=True, cmap="Purples")
+    ax2.set_title("Number of Multi-Country Clinical Trials Recruiting by Country")
+    ax2.annotate(f'Source: EBM Data Lab COVID-19 TrialsTracker, %s' %
+                 date.today().strftime("%b-%d-%Y"),
+                 xy=(0, 0), xycoords="axes points")
+
+    plt.savefig(args.output_map + '.png', bbox_inches="tight")
+    plt.savefig(args.output_map + '.svg', bbox_inches="tight")
+
+    print(f'Wrote {args.output_map}.png and {args.output_map}.svg')
+
     # The placeholder will be replaced by the actual SHA-1 hash in separate
     # script after the updated image is committed
     ebm_stats['ebm_trials_figure'] = \
         f'https://github.com/greenelab/covid19-review/raw/$FIGURE_COMMIT_SHA/{args.output_figure}.svg'
-    
+
```

Comment:
```suggestion
ebm_stats['ebm_map_figure'] = \
        f'https://github.com/greenelab/covid19-review/raw/$FIGURE_COMMIT_SHA/{args.output_map}.svg'
```

To save the figure URL for later use in the manuscript, you should be able to do the above.  `$FIGURE_COMMIT_SHA` will be replaced when the `version-figures.sh` script on the external-resources branch runs.